python/requirements.txt

```diff
@@ -0,0 +1,5 @@
+soundfile==0.12.1
```

Comment:
I think this whole file should go because setup.py provides these requirements.

Reply:
We decided to move all deps to install_requires and make requirements just -e .